Sally7.Tests/RequestExecutor/JobPoolTests.cs

```diff
@@ -19,11 +19,11 @@ public async Task RentJobIdAsync_Throws_If_Disposed_And_Depleted()
     }
 
     [Fact]
-    public void ReturnJobId_Does_Not_Throw_If_Disposed()
+    public async Task ReturnJobId_Does_Not_Throw_If_Disposed()
     {
         // Arrange
         var sut = new JobPool(1);
-        var jobId = sut.RentJobIdAsync(CancellationToken.None).Result;
+        var jobId = await sut.RentJobIdAsync(CancellationToken.None);
```

Comment:
The invocation was sync because it completes in sync fashion, not sure if one of the analyzers was bugging you to change this to async / await, but I'll accept it either way.